pages/search/index.vue

```diff
@@ -1,23 +1,34 @@
 <template>
-  <b-container data-qa="search page">
-    <b-row>
-      <b-col>
-        <h1>{{ $t('search') }}</h1>
-      </b-col>
-      <SearchInterface
-        :per-row="4"
-      />
-    </b-row>
-  </b-container>
+  <div>
+    <NotificationBanner
```

Comment:
This should only be displayed when the feature toggle `ENABLE_LINKS_TO_CLASSIC` is enabled, as with on the homepage already. That is to enable us to trivially switch it off without needing to make code-level changes & deployments.